documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Each Corporate Overlord will contribute an $300 initial capital contribution
+to BIFFUD within three months of achieving Corporate Overlord status.
```

Comment:
Have we made a decision on this?  It might make sense to have everyone still pay the same amount and so begin with a higher operating budget.  This would serve as a way to keep people engaged - sort of like planning a trip and having people pay upfront so you don't drop out later, forcing everyone else to pay more.  Thoughts?